core/arch/arm/arm.mk

```diff
@@ -18,3 +18,10 @@ core-platform-cppflags += $(arm32-platform-cppflags)
 core-platform-cflags += $(arm32-platform-cflags)
 core-platform-aflags += $(arm32-platform-aflags)
 endif
+
+ifeq ($(CFG_WITH_PAGER),y)
+ifneq ($(CFG_DISABLE_CONCURRENT_EXEC),y)
```

Comment:
These checks are temporary, they will go away when the concurrent pager patches are merged.
The patches are more or less done it's just that I'd like to take this in small steps.